oak_remote_attestation_noninteractive/proto/v1/messages.proto

```diff
@@ -24,10 +24,23 @@ option java_package = "oak.session.noninteractive.v1";
 message PublicKeyInfo {
```

Comment:
Do you think that it could be called `Evidence` or `AttestationEvidence` ?

Reply:
Are you suggesting this to match the RATS terminology? 

I'd be happy to rename it to `AttestationEvidence`, if the `AttestationEvidence` in the RATS terminology is actually meant to contain this information and be exchanged between an untrusted component and the client.

Reply:
yeah
It's not only the Evidence
In RATS it will be a combination of `Evidence` and `Endorcements`
I think we can rename it into `AttestationEvidence` for now, until we decide whether we want to use the term Endorcements from RATS or Transparent Release